dvc/utils/strictyaml.py

```diff
@@ -261,12 +264,25 @@ def validate(
     path: Optional[str] = None,
     rev: Optional[str] = None,
 ) -> _T:
-    from voluptuous import MultipleInvalid
+    from voluptuous import MultipleInvalid, TypeInvalid
 
     try:
         return schema(data)
     except MultipleInvalid as exc:
-        raise YAMLValidationError(exc, path, text, rev=rev) from exc
+        # Drop errors for str interpolation values
```

Comment:
I am not comfortable doing this. We should skip schema validation until we resolve.